config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -896,6 +896,7 @@ presubmits:
         - --build=bazel
         - --cluster=
         - --extract=local
+        - --flush-mem-after-build=true
```

Comment:
pull-kubernetes-e2e-containerd-gce isn't our job - do you really want to change it?

Reply:
Right. This should be change for `pull-kubernetes-kubemark-e2e-gce-big`